locales/sv_SE/pcsx2_Main.po

```diff
@@ -1739,34 +1737,34 @@ msgid "No plugin loaded"
 msgstr "Inget InsticksProgram laddat"
 
 #: pcsx2/gui/MainFrame.cpp:808
+msgid "&Plugin Settings..."
+msgstr "InsticksInställningar ..."
+
+#: pcsx2/gui/MainFrame.cpp:809
+#, c-format
+msgid "Opens the %s plugin's advanced settings dialog."
+msgstr "Öppnar %s-InsticksProgrammets fördjupande InställningsDialog."
```

Comment:
```suggestion
msgstr "Öppnar %s-Insticksprogrammets avancerade Inställningsdialog."
```

Reply:
"fördjupande" is a pefectly usable True swedish word here - no need for the swenglish or latino-swedish "avancerade".